engine/engine-script

```diff
@@ -7,21 +7,21 @@ echo "BASH_VERSION=${BASH_VERSION}"
 
 test -e /etc/profile && . /etc/profile
 
-if [ ! -e /usr/local/bin/client-server-script-library ]; then
-    echo "ERROR: Could not location client-server-script-library"
+if [ ! -e /usr/local/bin/engine-script-library ]; then
+    echo "ERROR: Could not location engine-library"
```

Comment:
engine-script-library?

Reply:
yeah, odd naming...I failed to come up with anything better I suppose:

Reply:
https://github.com/k-rister/rickshaw/tree/dev-kmr/engine